deployment/values.yaml

```diff
@@ -9,4 +9,6 @@ mongodb:
   persistent: true
   version: 3.6
 
+namespaces: omp-frontend
```

Comment:
Is this the only place this is set? If overriding would like to not have omp-frontend by default but instead `lodestar-frontend` or whatever the new value will be.

Reply:
Eventually, this should be a list of all of the namespaces where LodeStar components live.  This is currently the only place this is defined.  I also thought about putting this in the development repository instead.  Not sure which is the best place.  The only reason it is currently `omp-frontend` is that is what is in the `test/integration/prod` environments and I want it to work when we deploy.  We will have to change it to `lodestar-frontend` as soon as that changes.  Otherwise, no components will be found to validate it is working.